torchci/test/stripApprovals.test.ts

```diff
@@ -0,0 +1,23 @@
+import nock from "nock";
+import { Probot } from "probot";
+import myProbotApp, { CIFLOW_TRUNK_LABEL } from "../lib/bot/acceptBot";
+import { handleScope, requireDeepCopy } from "./common";
+import * as utils from "./utils";
+nock.disableNetConnect();
+
+describe("strip approvals bot", () => {
+  let probot: Probot;
+
+  beforeEach(() => {
+    probot = utils.testProbot();
+    probot.load(myProbotApp);
+  });
+
+  test("Do nothing on normal contributor", async () => {
```

Comment:
not a requested reviewer but I think this test is misnamed as the author is huy who has write permissions, and could you add a test for when the author doesn't have write permissions?